lib/haml_lint/tree/tag_node.rb

```diff
@@ -208,10 +222,6 @@ def script
 
     private
 
-    def parsed_attributes
```

Comment:
Since this is a private method, I felt that I could override it with the thin `ParsedRuby` wrapper without issue.